lib/logger.js

```diff
@@ -7,6 +7,8 @@ const { EOL } = require('os');
 function paint(level) {
```

Comment:
This should probably be renamed to something like "text" or "label" now because stack or data are not logging levels

Reply:
:+1: You are right. I'll correct it.

Reply:
Done.